pkg/assert/assert.go

```diff
@@ -75,5 +78,5 @@ func checkStepResults(t *testing.T, steps []v1.StepState, resultName string) {
 			t.Fatalf("Step result '%s' in step '%s' is empty", resultName, step.Name)
```

Comment:
I believe the usage of Fatalf is in this file is likely not what we want.

Please see go/go-style/best-practices#t-fatal.

Reply:
see your points. as the part of the verification flows, here we should "error" it to keep the test run for the rest of steps right.